tuxedo-core/src/executive.rs

```diff
@@ -164,18 +165,39 @@ impl<
         );
 
         // Re-do the pre-checks. These should have been done in the pool, but we can't
-        // guarantee that foreign nodes to these checks faithfully, so we need to check on-chain.
-        let valid_transaction = Self::validate_tuxedo_transaction(&transaction)?;
+        // guarantee that foreign nodes do these checks faithfully, so we need to check on-chain.
+        let prelim_valid_transaction = Self::validate_tuxedo_transaction(&transaction)?;
 
         // If there are still missing inputs, we cannot execute this,
         // although it would be valid in the pool
         ensure!(
-            valid_transaction.requires.is_empty(),
+            prelim_valid_transaction.requires.is_empty(),
             UtxoError::MissingInput
         );
 
+        // Process the accumulator
+        let acc_key = C::Accumulator::key_path(
+            <C::Accumulator as Accumulator>::ValueType::decode::<&[u8]>(&mut &Vec::<u8>::new()[..])
+                .expect("TODO!"),
+        ); // TODO what should this key be?
```

Comment:
Do we have the answer to this question?

Reply:
The `acc_key` should be something that one of the pieces' accumulators returns. For example `imbalance` from the money piece or `stub_acc` from the `()` implementation.

The prefixed one right below would then be `:accumulator:imbalance`. The point of prefixing them is that we can kill them all in one fell swoop at the end of the block.